package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "findthemasks",
+  "description": "findthemasks",
+  "version": "1.0.0",
+  "license": "MIT",
```

Comment:
Not a `node.js` buff, but in case this option is still used: Would you mind defining a node version requirement in here as well? Good to have documented for reference. Similar to how express itself does it, e.g.

```json
"engines": {
    "node": ">= 0.10.0"
  }
```

c.f. https://github.com/expressjs/express/blob/master/package.json#L82

Reply:
p.s. I only ask for this since I along with many other people may have widely variant versions of `node.js` running locally on their systems and may need to upgrade or switch (e.g. via `nvm`) in order to run the app, in case any deeper dependencies end up needing a newer version.

Reply:
+1 good point, I'd vote for latest Long Term Support version (v12) but I'd be fine with v10 too

Reply:
@nburt set it to > 12.x.x.